syncany-lib/src/test/java/org/syncany/tests/util/TestClient.java

```diff
@@ -255,7 +255,7 @@ public void moveFile(String fileFrom, String fileTo) throws Exception {
 			}
 		}
 		catch (Exception e) {
-			throw new Exception("Move failed: " + fileFrom + " --> " + fileTo, e);
```

Comment:
Is this intentional?

Reply:
No.

Reply:
Actually, I'll add a test around that, the build should have failed.

Reply:
There's a couple of other places where the --> to --&gt; replace was done in non-javadoc, but beyond this one it was only comments. You could revert those as well, but it's not a big deal to me. I think a test here might be a little overkill, as it is indeed testcode.

Reply:
Yep, you're right, it would be overkill.  I thought I had accidentally removed the Exception.

Reply:
I reverted all the non-Javadoc replacements of `>` into `&gt;`.